src/features/singleRight/delegate/ReceiptPage/ActionBarSection/ActionBarSection.tsx

```diff
@@ -0,0 +1,196 @@
+
+  let mostFailedIndex = -1;
+  let mostFailedDelegations = -1;
+
+  const firstSuccesfulIndex = delegations.findIndex((pd: ProcessedDelegation) => {
+    const failedDelegations = pd.bffResponseList?.filter(
+      (data: DelegationResponseData) => data.status !== BFFDelegatedStatus.Delegated,
+    );
+    return failedDelegations?.length === 0;
+  });
+
+  // logic for finding the delegation with the most failed Delegations
+  delegations.forEach((pd, index) => {
+    const failedDelegations = pd.bffResponseList?.filter(
+      (data: DelegationResponseData) => data.status !== BFFDelegatedStatus.Delegated,
+    );
+
+    const numFailedDelegations = failedDelegations?.length || 0;
+
+    if (numFailedDelegations > mostFailedDelegations) {
+      mostFailedDelegations = numFailedDelegations;
+      mostFailedIndex = index;
+    }
+  });
+
+  const actionBars = delegations
+    .map((pd: ProcessedDelegation, index: number) => {
+      const failedDelegations = pd.bffResponseList?.filter(
+        (data: DelegationResponseData) => data.status !== BFFDelegatedStatus.Delegated,
+      );
+
+      const successfulDelegations = pd.bffResponseList?.filter(
+        (data: DelegationResponseData) => data.status !== BFFDelegatedStatus.NotDelegated,
+      );
+
+      const numFailedDelegations = failedDelegations?.length || 0;
+
+      const additionalText = () => {
+        if (numFailedDelegations > 0) {
+          return (
+            <div className={classes.additionalTextContainer}>
+              <ErrorMessage className={classes.additionalText}>
+                {numFailedDelegations +
+                  '/' +
+                  pd.bffResponseList?.length +
+                  ' ' +
+                  t('common.failed_lowercase')}
+              </ErrorMessage>
+              <ExclamationmarkTriangleIcon className={classes.warningIcon} />
+            </div>
+          );
+        } else {
+          return undefined;
+        }
+      };
+
+      const dangerAlert = () => {
+        return (
+          <div className={classes.alertContainer}>
+            {numFailedDelegations > 0 && (
+              <Alert severity='danger'>
+                <Heading
+                  size={'xsmall'}
+                  level={2}
+                  spacing
+                >
+                  {t('single_rights.woops_something_went_wrong_alert')}
+                </Heading>
+                <Paragraph spacing>{t('single_rights.some_failed_technical_problem')}</Paragraph>
+                <Heading
+                  size={'xxsmall'}
+                  level={3}
+                >
+                  {t('single_rights.these_rights_were_not_delegated')}
+                </Heading>
+                <div className={classes.chipContainer}>
+                  <Chip.Group size='small'>
+                    {failedDelegations?.map((failedRight, innerIndex) => {
+                      return (
+                        <Chip.Toggle key={`failed-${index}-${innerIndex}`}>
```

Comment:
No it should't since failed actions get failed in front of key and succesful get successful

![image](https://github.com/Altinn/altinn-access-management-frontend/assets/31471142/37f7ee89-28cf-452c-8ae5-8fe01f3186ca)

Reply:
You're right, of course. My bad 😅